sources/forms-docker-compose.yml

```diff
@@ -0,0 +1,36 @@
+version: '3.7'
+services:
+  mongo:
+    image: mongo:4.1
+    restart: always
+    volumes:
+      - mdb-data:/data/db
+    environment:
+      MONGO_INITDB_ROOT_USERNAME:
+      MONGO_INITDB_ROOT_PASSWORD:
+  
+  formio:
+    build:
+      context: ./packages/forms
+      args:
+        - FORMIO_SOURCE_REPO_URL=${FORMIO_SOURCE_REPO_URL}
+        - FORMIO_SOURCE_REPO_TAG=${FORMIO_SOURCE_REPO_TAG}
+    # The app will restart until Mongo is listening
+    restart: always
+    links:
+      - mongo
+    ports:
+      - "3001:3001"
```

Comment:
The port must come from environment value right?

Reply:
It should be, but as this is for local deployment, harcoaded it. As we are referring the same in the other docker-compose.yml as formio:3001 for the forms_url parameter.